src/test/java/io/stargate/sgv2/jsonapi/api/v1/util/DataApiResponseValidator.java

```diff
@@ -233,4 +237,13 @@ public DataApiResponseValidator hasDocuments(int size) {
   public DataApiResponseValidator hasDocumentInPosition(int position, String documentJSON) {
     return body("data.documents[%s]".formatted(position), jsonEquals(documentJSON));
   }
+
+  public DataApiResponseValidator hasIndexes(String... indexes) {
+    return body("status.indexes", hasSize(indexes.length))
+        .body("status.indexes", containsInAnyOrder(indexes));
+  }
+
+  public DataApiResponseValidator shouldNotHaveIndexes(String... indexes) {
+    return body("status.indexes", not(containsInAnyOrder(indexes)));
```

Comment:
Does this check that none of indexes is contained (intent) or that not all of them are contained?
I think it would be latter and if so would instead need to either iterate over individual ones, or use different predicate ("constainsNoneOf()" if such exists?)